src/dispatch/alembic/versions/cfd62f719c84_.py

```diff
@@ -0,0 +1,29 @@
+"""empty message
+
+Revision ID: cfd62f719c84
+Revises: 5693756fd192
+Create Date: 2020-05-26 20:17:47.707953
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "cfd62f719c84"
+down_revision = "5693756fd192"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("incident_type", "plugin_metadata")
+    op.add_column("incident_type", sa.Column("plugin_metadata", sa.JSON(), nullable=True))
```

Comment:
Why not using [`alter_column()`](https://alembic.sqlalchemy.org/en/latest/ops.html?highlight=insert#alembic.operations.Operations.alter_column)? Aren't we going to lose the data if we drop the column?

Reply:
I think he did this to ensure any data that existed gets dropped since the format of the JSON has changed.